python/res/enkf/jobs/scaling/scaling_job.py

```diff
@@ -219,7 +219,8 @@ def has_data(observations, keys, ensamble_size, storage):
     Checks that all keys have data and returns a list of error messages
     """
     error_msg = "Key: {} has no data"
-    active_mask = BoolVector(True, ensamble_size)
+    active_realizations = storage.realizationList(RealizationStateEnum.STATE_HAS_DATA)
+    active_mask = BoolVector.createFromList(ensamble_size, active_realizations)
```

Comment:
Could I have a test that fails on the previous setup and now succeeds? :)

Reply:
:+1: